standard/docs/en/schema/merging.md

```diff
@@ -1,30 +1,31 @@
-# Merging 
+# Merging
 
-In OCDS, merging involves combining individual [releases](../getting_started/releases_and_records.md) of data during a contracting process into a [record](../getting_started/releases_and_records.md) which provides an overview of the current state of that process. Including a versioned release as part of the merged record provides structured information on the history of the process: showing when updates were made, or values were changed. 
+An OCDS [record](../getting_started/releases_and_records.md) aggregates all the releases available for a contracting process at a given time, and optionally includes:
 
-<div class="example hint" markdown=1>
+* a compiled release, which expresses the current state of the contracting process, by showing only the most recent field values
+* a versioned release, which expresses all historical states of the contracting process, by showing all the field values over time
 
-<p class="first admonition-title">Worked Example</p>
+**Merging** is the process of combining individual releases into a compiled or versioned release, described in more detail below.
 
-A publisher provides a tender release on 1st January, with a planned contract value of $1000.
+The versioned release makes it easy to see how field values have changed over time: for example, how contract values have been modified or how milestones have been re-scheduled during implementation.
 
-On 5th February, the publisher provides an amended tender release updating the planned contract value to $2000.
+<div class="example hint" markdown=1>
 
-After assessing bids, it is decided to award the contract in two lots.
+<p class="first admonition-title">Worked Example</p>
```

Comment:
It would be good to expand the example (or add another example) demonstrating null-ing for values, objects and arrays.

I don't think we need to do that now, but I added it to the list of suggested examples for OCDS 1.1.5